### PR TITLE
Fixing Django 1.10 warnings and enable ldap attributes for iter_users

### PR DESCRIPTION
**Pull request by frennkie**
Replace NoArgsCommand as it is deprecated as of Django 1.10.

```
./manage.py ldap_sync_users
/home/robbie/work/nmap_tool/venv/lib/python3.4/site-packages/django/core/management/base.py:577: RemovedInDjango110Warning: NoArgsCommand class is deprec
ated and will be removed in Django 1.10. Use BaseCommand instead, which takes no arguments by default.
  RemovedInDjango110Warning
```

I took the directly from a commit made to your watson repository.. :-)

### CODE REVIEW

File: django_python3_ldap/ldap.py [hunk 1]
```diff
@@ -68,6 +68,7 @@ def iter_users(self):
             search_filter = format_search_filter({}),
             search_scope = ldap3.SEARCH_SCOPE_WHOLE_SUBTREE,
             attributes = ldap3.ALL_ATTRIBUTES,
+            get_operational_attributes = True,
```
Comment (etianen) [inline django_python3_ldap/ldap.py]: What does this line do? Seems unrelated to the rest of the pull request.

**Comment by etianen** [comment]
Looks good, but I could do with some clarification on my line comment. :)

**Comment by frennkie** [comment]
I accidentially put two seperate commits into this PR.  
The "long" description for line 71 is in that commit: https://github.com/etianen/django-python3-ldap/pull/41/commits/60985ad25a058839c798d8dea691a0fe80856660

Thanks

File: django_python3_ldap/ldap.py (hunk 1 above)
Comment (etianen) [inline django_python3_ldap/ldap.py]: Understood, thanks!
